packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -639,6 +639,9 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
       checked: item.checked,
       icon: item.icon,
       iconProps: item.iconProps,
+      onMouseEnter: this._onItemMouseEnter.bind(this, item, true /* isSplitButtonPrimary */),
+      onMouseLeave: this._onMouseItemLeave.bind(this, item),
+      onMouseMove: this._onItemMouseMove.bind(this, item, true /* isSplitButtonPrimary */),
```

Comment:
>.bind [](start = 40, length = 5)

Question a bit unrelated to the CR, do we not prefer to tag the methods with @autobind instead of using .bind, this way we dont have to remember to do it? I ask because I think I have seen both ways in the project and wondering if there is a preference

Reply:
No, the usage of autobind has been removed from fabric because of the bloat that came with it. Ideally you want to use functions as variables instead, but in this case since we are passing extra data to the callback we need to use bind